src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -30,12 +30,19 @@ internal class RequestTracingConstants
         public const string KeyVaultConfiguredTag = "UsesKeyVault";
         public const string KeyVaultRefreshConfiguredTag = "RefreshesKeyVault";
         public const string ReplicaCountKey = "ReplicaCount";
+        public const string FeatureFlagUsesTelemetryTag = "FFTelemetry";
+        public const string FeatureFlagUsesSeedTag = "FFSeed";
+        public const string FeatureFlagMaxVariantsKey = "MaxVariants";
+        public const string FeatureFlagUsesVariantConfigurationReferenceTag = "FFConfigRef";
+        public const string FeaturesKey = "Features";
```

Comment:
Do we want to scope this to feature flags only? If so, it probably makes more sense to have "FFFeatures". On the other hand, we could drop the FF from others, so we will have, for example, FFFeatures=Seed+Telemetry+ConfigRef.

Reply:
It sounds reasonable to me. But that also means we have non-feature flag related items we would have `Features=` and `FFFeatures=`

Reply:
Since we don't currently have anything to under `Features=`, should I just add the constant but not use it yet or is there something we want to move into this section?

Reply:
We're not using `Features=` at the moment. There's no reason to have it.